mobile/src/main/java/ch/epfl/sdp/mobile/ui/features/home/StatefulHome.kt

```diff
@@ -0,0 +1,66 @@
+package ch.epfl.sdp.mobile.ui.features.home
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.navigation.NavBackStackEntry
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.currentBackStackEntryAsState
+import androidx.navigation.compose.rememberNavController
+import ch.epfl.sdp.mobile.data.api.AuthenticationApi
+import ch.epfl.sdp.mobile.ui.features.profile.StatefulProfileScreen
+import ch.epfl.sdp.mobile.ui.features.social.StatefulFollowingScreen
+
+/** The route associated to the social tab. */
+private const val SocialRoute = "social"
```

Comment:
Modification : variable name should be uppercase

```suggestion
private const val SOCIAL_ROUTE= "social"
```

Reply:
I'm not sure about that one, since the AndroidX repository seems to accept both naming conventions for internal / private usage (see https://github.com/androidx/androidx/search?l=Kotlin&q=const).

However, all of the public Compose API seems to use PascalCase, since it tries to avoid distinguishing between `public const val` and `const val` values. To keep it consistent with the Compose API, I'd suggest we keep PascalCase for now, but we can discuss this with the team in the next meeting. Does this sound good to you ?